Builds/CMake/RippledSettings.cmake

```diff
@@ -3,7 +3,18 @@
 #]===================================================================]
 
 option (assert "Enables asserts, even in release builds" OFF)
-option (unity "Creates a build based on unity sources. This is the default" ON)
+
+option (unity "Creates a build using UNITY support in cmake. This is the default" ON)
+if (unity)
+  if (CMAKE_VERSION VERSION_LESS 3.16)
+    message (WARNING "unity option only supported for with cmake 3.16+ (please upgrade)")
+    set (unity OFF CACHE BOOL "unity only available for cmake 3.16+" FORCE)
```

Comment:
We might want to give an error here instead. They asked for unity build, and we can't give it to them. They can always make a non-unity build themselves. What do you think? I do see the arguments to reverting to a non-unity build as you do here, so if you prefer that I'm OK with that too.

Reply:
if `unity` were NOT the default, I might agree....as it is, I'd hate to fail the configure step for default configurations for anyone using cmake less than 3.16.